src/components/anchor-navigation/anchor-navigation.spec.tsx

```diff
@@ -191,64 +183,25 @@ describe("AnchorNavigation", () => {
           element: wrapper.find(".focusableContent").at(index).getDOMNode(),
           options: { block: "start", inline: "nearest", behavior: "smooth" },
         });
+
+        expect(wrapper.find(Content).at(index)).toBeFocused();
       }
     );
 
-    it("when key is pressed on navigation item and target section contains a focusable element, focus on that element", () => {
-      const sectionIndex = 0;
+    it("does not alter the tabindex of the container if it was already focusable", () => {
```

Comment:
I'd rather not as that's just an ugly implementation detail that I'd rather do without (but don't think there's a way to). The important thing is to programatically focus the container - which is tested for elsewhere - and setting tabIndex to -1 is just a means to that end. This particular test was I think mainly for coverage but also because it's good to ensure that we don't accidentally make things not tabbable that should have been.

Will happily discuss and/or wait for other comments.

Reply:
I agree with @robinzigmond here. As long as we have a test that programatically focuses the container, that _should_ be enough. Famous last words 🤣 
If we end up having an issue in the future because of this lack of test, we'll take it as a lesson in life. If I'm also missing the significance of having a test that covers containers without a tab index, I'm happy to be shown the light.